_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,419 @@
+	<input type="button" onclick="stop()" value="Stop updates" />
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		var updates
+		function start() {
+			updates = setInterval(change, 3000)
+		}
+
+		function stop() {
+			clearInterval(updates)
+		}
+	</script>
+</body>
+```
+
+#### Passed Example 2
+
+The text node automatically updates every 3 seconds after the page completes loading. A mechanism is available to pause and resume the automatic updates.
+
+```html
+<body onload="toggle()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" id="control" onclick="toggle()" value="Pause updates" />
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		var updates
+		var updating = false
+
+		function toggle() {
+			var control = document.getElementById('control')
+			if (updating) {
+				control.value = 'Resume updates'
+				updating = false
+				clearInterval(updates)
+			} else {
+				control.value = 'Pause updates'
+				updating = true
+				updates = setInterval(change, 3000)
+			}
+		}
+	</script>
+</body>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The text node automatically updates every 3 seconds after the page completes loading. There is no mechanism to stop or pause the automatic updates.
+
+```html
+<body onload="start()">
+	<p>Random number: <span id="target">1</span></p>
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		function start() {
+			updates = setInterval(change, 3000)
+		}
+	</script>
+</body>
+```
+
+#### Failed Example 2
+
+The text node automatically updates every 3 seconds after the page completes loading. The mechanism to stop the automatic updates is not visible.
+
+```html
+<body onload="start()">
+	<p>Random number: <span id="target">1</span></p>
+	<input type="button" onclick="stop()" value="Stop updates" style="position: absolute; top: -9999px; left: -9999px;" />
+
+	<script type="text/javascript">
+		function change() {
+			var target = document.getElementById('target')
+			var number = Math.floor(Math.random() * 1000)
+			target.innerText = number
+		}
+
+		var updates
+		function start() {
+			updates = setInterval(change, 3000)
+		}
+
+		function stop() {
+			clearInterval(updates)
+		}
+	</script>
+</body>
+```
+
+#### Failed Example 3
+
+The text node automatically updates every 3 seconds after the page completes loading. The mechanism to stop the automatic updates is not included in the accessibility tree.
+
+```html
+<body onload="start()">
+	<p>Random number: <span id="target">1</span></p>
+	<div onclick="stop()" aria-hidden="true">
```

Comment:
This `div` should actually be a span to avoid the clickable area spreading over the full line.

Actually, the `div` and the `span` should be merged into a single element. There is nothing added to the example by having the `onclick` property on another node than the text.